smaps/src/main/webapp/index.html

```diff
@@ -54,6 +54,11 @@
     <div class="row justify-content-md-center">
         <p id="error-message" name="error-message"></p>
     </div>
+    <div class="row justify-content-md-center">
+        <form action="/fileupload" method="POST" enctype="multipart/form-data">
+            <input type="submit" name="example-btn" id="example-btn" value="EXAMPLE MODE">
```

Comment:
Why does this button end up as one <input ...> span but the other is two?

Reply:
This one doesn't need a separate input to accept a file because the user isn't picking one, so it just needs a button to say it's doing example mode.